lib/eth/EthRpc.js

```diff
@@ -250,7 +250,12 @@ class EthRPC {
         .sendSignedTransaction(rawTx)
         .once('transactionHash', resolve)
         .once('error', reject)
-        .catch(err => reject(err));
+        .catch((err) => {
+          if (err.message.includes('nonce')) {
```

Comment:
maybe overkill, but could we be a bit more defensive and check for `err.message` as well?

Reply:
Fixed